src/components/Tooltip/Tooltip.tsx

```diff
@@ -40,12 +40,11 @@ import {
   TooltipLabel,
   useTooltip,
 } from "./useTooltip";
-import { XOR } from "ts-xor";
 
 // Unfortunately Omit doesn't distribute nicely over sum types, so we have to
 // piece together the useTooltip options type by hand
 type TooltipProps = Omit<CommonUseTooltipProps, "isTriggerInteractive"> &
-  XOR<TooltipLabel, TooltipDescription> & {
+  (TooltipLabel | TooltipDescription) & {
```

Comment:
`ts-xor` really makes it impossible to use in some context, especially stories, so I removed it